spaceR/spaceR/ViewModels/LoginViewModel.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  LoginViewModel.swift
+//  SpaceR
+//
+//  Created by Kieran Woodrow on 2022/03/13.
+//
+
+import Foundation
+import UIKit
+
+class LoginViewModel {
+    
+    private var user: [User] = []
+    // swiftlint:disable force_cast
+    private let coreDataPersistantObject = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
```

Comment:
Rather use an optional cast 😄

Reply:
Okay, i changed it. Please have a look if it better